tests/test_sensors.py

```diff
@@ -117,6 +117,8 @@ def test_sensors(
     if not habitat_sim.cuda_enabled and gpu2gpu:
         pytest.skip("Skipping GPU->GPU test")
 
+    sim.close()
```

Comment:
We can do tuse `with habitat_sim as sim` to make the scope more clear here

Reply:
That will require re-factoring the tests to no share an instance of the simulator.  Which we should do, but best not in this PR imo.